misc/windows-iam/Setup_Iam.ps1

```diff
@@ -0,0 +1,32 @@
+# Copyright 2014-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved
```

Comment:
The networking changes required for IAM roles to work on Windows are significantly different from that on Linux.  We will need to document how this works in more detail, but may not get to it before merging this.